definitions/ext-pixie_mysql/golden_metrics.yml

```diff
@@ -0,0 +1,18 @@
+throughput:
```

Comment:
Just leaving this comment FYI regarding customer experience: The golden metrics on the Infra type are looking differently https://github.com/newrelic/entity-definitions/blob/main/definitions/infra-mysqlnode/golden_metrics.yml

Would you like to change it or shall we merge for testing ?

Reply:
So for this set of metrics, we currently do not capture this data and for many instances we cannot capture the data (ie if the MySQL instance is hosted by a cloud provider). Had a discussion with @nserrino and I think sticking with these metrics to be consistent with SERVICE definitions is our preference.